superset/tasks/utils.py

```diff
@@ -133,7 +133,7 @@ def fetch_csrf_token(
             data = json.loads(body)
             res = {"X-CSRF-Token": data["result"]}
             if session_cookie is not None:
-                res["Cookie"] = session_cookie
+                res["Cookie"] = f"session={session_cookie}"
```

Comment:
Good timing, I just stumbled on this bug yesterday in 4.1.1.  One minor tweak though - could you use the `session_cookie_name` param like so? 
```suggestion
                res["Cookie"] = f"{session_cookie_name}={session_cookie}"
```
In my local testing it worked as expected since it defaults to `session`.

Reply:
Thanks a lot!. This is a good suggestion